superset/assets/src/dashboard/util/publishSubscriberUtil.js

```diff
@@ -30,17 +32,62 @@ export function getUniqueActionsForSlice(slice) {
 export function getModalSliceIDFor(publishSubscriberMap, publisherId) {
     if (publishSubscriberMap && publishSubscriberMap.hasOwnProperty('subscribers')) {
         const subscribers = Object.values(publishSubscriberMap.subscribers);
-        var item = _.find(subscribers, function (item) {
+        var item = find(subscribers, function (item) {
             return ((item.actions.indexOf(USE_AS_MODAL) > -1) && (Object.keys(item.linked_slices).indexOf(publisherId) != -1));
         });
         return item ? item.id : undefined;
     }
     return undefined
 }
 
+export function getSlicesWithSubHeader(subscribers, chartId, filters) {
+    let subHeader = '';
```

Comment:
function name should be like getSubheaderForSlice

Reply:
Done